packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -915,23 +915,23 @@ async function determinePackageBasedOrIntegratedOrStandalone(): Promise<
     {
       type: 'autocomplete',
       name: 'workspaceType',
-      message: `Package-based monorepo, integrated monorepo, or standalone project?`,
+      message: 'npm/yarn monorepo, Nx monorepo, or standalone project?',
```

Comment:
I feel saying Nx monorepo will be weird for first time users.

So the former is not an Nx monorepo? :thinking: I know what you mean but... it is also an Nx monorepo because it works in conjunction to npm/yarn/pnpm